src/v/crypto/tests/test_openssl_conf.cnf.in

```diff
@@ -1,6 +1,6 @@
 config_diagnostics = 1
 openssl_conf = openssl_init
-.include @REDPANDA_DEPS_INSTALL_DIR@/openssl/fipsmodule.cnf
+.include @REDPANDA_DEPS_INSTALL_DIR@/etc/ssl/fipsmodule.cnf
```

Comment:
question: Does this depend on vtools changes?

Reply:
yes